qiime2/sdk/plugin_manager.py

```diff
@@ -93,18 +93,38 @@ def _integrate_plugin(self, plugin):
     # databases rollback/commit
 
     @property
+    def importable_formats(self):
+        """Return set of formats that are importable.
```

Comment:
What's the advantage of returning it as a dict?  In its current form you can pretty easily get the name using `thing_in_the_set.format.__name__`, so it seems to me like putting that in a dict results in redundancy.  Am I missing something?  Or is it just more elegant in a dict?